src/main/java/frc/robot/Robot.java

```diff
@@ -28,25 +27,14 @@
  */
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
-
-
-
   final DriveBase m_driveSubsystem = new DriveBase();
-
-  final Gyro gyro = new Gyro();
```

Comment:
Removed a bunch of whitespace cuz it annoys me but also removed the existing Gyro class cuz we don't have a gyro yet and if/when we do, Owen will expertly implement the new gyro code and we also likley won't be using a NavX gyro.

Reply:
we should keep the gyro but the rest of the code should be good

Reply:
Gyro was put back in.